numpy/core/_add_newdocs.py

```diff
@@ -714,6 +714,7 @@
 
     """))
 
+
```

Comment:
I am pretty sure the warnings genetated by the `Example` sections here are bugs. The problem is that the docstring attributes for `broadcast.nd`, `broadcast.ndim`, and `broadcast.numiter` are being [inlined](https://docs.scipy.org/doc/numpy-1.16.1/reference/generated/numpy.broadcast.html) rather than linked to, but only for these three attributes.

Reply:
Any idea why only those are being inlined?

Reply:
No. 

It is complicated to find, since the real error is not the warning, but that these attributes are being handled differently by the autsummary mechanism.